zipkin-storage/cassandra-v1/src/main/java/zipkin2/storage/cassandra/v1/SelectDependencies.java

```diff
@@ -75,24 +75,13 @@ public SelectDependencies clone() {
     return new SelectDependencies(factory, days);
   }
 
-  enum ConvertDependenciesResponse implements Mapper<ResultSet, List<DependencyLink>> {
-    INSTANCE;
-
-    @Override
-    public List<DependencyLink> map(ResultSet rs) {
-      List<DependencyLink> unmerged = new ArrayList<>();
-      for (Row row : rs) {
-        ByteBuffer encodedDayOfDependencies = row.getBytes("dependencies");
-        for (DependencyLink link : Dependencies.fromThrift(encodedDayOfDependencies).links()) {
-          unmerged.add(link);
-        }
-      }
-      return DependencyLinker.merge(unmerged);
-    }
-
-    @Override
-    public String toString() {
-      return "MergeDependencies";
+  @Override
+  public List<DependencyLink> map(ResultSet rs) {
```

Comment:
@devinsba this is an example of merging for dependency links. we can do it sooner or later in dynamodb